translations/ja.json

```
@@ -160,48 +160,48 @@
 	"Enabled addons: {{.addons}}": "有効なアドオン: {{.addons}}",
-	"Enables the addon w/ADDON_NAME within minikube. For a list of available addons use: minikube addons list ": "",
-	"Enabling '{{.name}}' returned an error: {{.error}}": "'{{.name}}' を有効にする際にエラーが発生しました。{{.error}}",
+	"Enables the addon w/ADDON_NAME within minikube. For a list of available addons use: minikube addons list ": "minikube 内で ADDON_NAME アドオンを有効化します。利用可能なアドオン一覧は、minikube addons list を使用してください",
+	"Enabling '{{.name}}' returned an error: {{.error}}": "'{{.name}}' 有効化がエラーを返しました: {{.error}}",
 	"Enabling dashboard ...": "ダッシュボードを有効化しています...",
-	"Ensure that CRI-O is installed and healthy: Run 'sudo systemctl start crio' and 'journalctl -u crio'. Alternatively, use --container-runtime=docker": "",
-	"Ensure that Docker is installed and healthy: Run 'sudo systemctl start docker' and 'journalctl -u docker'. Alternatively, select another value for --driver": "",
-	"Ensure that the required 'pids' cgroup is enabled on your host: grep pids /proc/cgroups": "",
-	"Ensure that the user listed in /etc/libvirt/qemu.conf has access to your home directory": "",
-	"Ensure that you are a member of the appropriate libvirt group (remember to relogin for group changes to take effect!)": "",
-	"Ensure that your value for HTTPS_PROXY points to an HTTPS proxy rather than an HTTP proxy": "",
-	"Ensure the tmp directory path is writable to the current user.": "",
-	"Ensure you have at least 20GB of free disk space.": "",
-	"Ensure your {{.driver_name}} is running and is healthy.": "",
-	"Environment variables to pass to the Docker daemon. (format: key=value)": "Docker デーモンに渡す環境変数（形式: Key=Value）",
-	"Environment variables to pass to the build. (format: key=value)": "",
-	"Error code docs have been saved at - {{.path}}": "",
-	"Error creating minikube directory": "minikube のディレクトリ作成中にエラーが発生しました",
+	"Ensure that CRI-O is installed and healthy: Run 'sudo systemctl start crio' and 'journalctl -u crio'. Alternatively, use --container-runtime=docker": "CRI-O がインストール済みで正常であることを確認してください: 'sudo systemctl start crio' と 'journalctl -u crio' を実行してください。または、--container-runtime=docker を使用してください",
+	"Ensure that Docker is installed and healthy: Run 'sudo systemctl start docker' and 'journalctl -u docker'. Alternatively, select another value for --driver": "Docker がインストール済みで正常であることを確認してください: 'sudo systemctl start docker' と 'journalctl -u docker' を実行してください。または、--driver に別の値を選択してください",
+	"Ensure that the required 'pids' cgroup is enabled on your host: grep pids /proc/cgroups": "必要な 'pids' がこのホスト上で有効であることを確認してください: grep pids /proc/cgroups",
+	"Ensure that the user listed in /etc/libvirt/qemu.conf has access to your home directory": "/etc/libvirt/qemu.conf にリストされたユーザーがあなたのホームディレクトリーにアクセス可能であることを確認してください",
+	"Ensure that you are a member of the appropriate libvirt group (remember to relogin for group changes to take effect!)": "あなたが適切な libvirt グループメンバーの 1 人であることを確認してください (グループ変更が効果を発揮するための再ログインをお忘れなく)",
```

```suggestion
	"Ensure that you are a member of the appropriate libvirt group (remember to relogin for group changes to take effect!)": "あなたが適切な libvirt グループメンバーの 1 人であることを確認してください (グループ変更を有効にするには再ログインをお忘れなく!)",
```
"有効にするには“ is more easier to understand I think.

---

applied.